src/sources/forus-webshop/js/angular-1/components/ProductsComponent.js

```diff
@@ -145,6 +145,20 @@ let ProductsComponent = function(
         $ctrl.updateFiltersUsedCount();
     };
 
+    $scope.scrolled = false;
+    var prevScrollpos = window.pageYOffset;
+
+    $scope.setScrolled = function() {
```

Comment:
@dev-rminds this setScrolled function is aswell defined in src/sources/forus-webshop/js/angular-1/components/ProvidersComponent.js ; probably this can be refactored abit